internal/access/identity_test.go

```diff
@@ -185,20 +185,20 @@ func TestUpdateIdentityInfoFromProvider(t *testing.T) {
 	}
 
 	t.Run("a revoked OIDC session revokes access keys created by provider login", func(t *testing.T) {
-		_, err = data.CreateProviderUser(db, provider, rCtx.Authenticated.User)
+		user, err := data.CreateProviderUser(db, provider, rCtx.Authenticated.User)
 		assert.NilError(t, err)
 		oidc := &fakeOIDCImplementation{UserInfoRevoked: true}
 
 		toBeRevoked := &models.AccessKey{IssuedFor: rCtx.Authenticated.User.ID, ProviderID: provider.ID}
-		_, err := data.CreateAccessKey(db, toBeRevoked)
+		_, err = data.CreateAccessKey(db, toBeRevoked)
```

Comment:
I'm confused about how this compiled in the old version.

Reply:
Oh ya, this is surprising! I think it's this: https://go.dev/play/p/huEC6yz8pur

There is an `err` in the parent scope, so it's fine to do `err = ...`, but it's also fine to do `err := ...` once in the new scope, even though it comes after assigning to `err` of the parent scope.